test/item_repository_test.rb

```diff
@@ -138,9 +138,9 @@ def test_find_all_by_merchant_id
   end
 
   def test_find_all_by_created_at
-    output_1 = item_repository.find_all_by_created_at(DateTime.parse('2012-03-27 14:53:59 UTC')).count
+    output_1 = item_repository.find_all_by_created_at(Date.parse('2012-03-27 14:53:59 UTC')).count
     expected_1 = 10
-    output_2 = item_repository.find_all_by_created_at(DateTime.parse('2015-03-27 14:53:59 UTC'))
+    output_2 = item_repository.find_all_by_created_at(Date.parse('2015-03-27 14:53:59 UTC'))
```

Comment:
Line is too long. [92/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.